samples/Nancy.Demo.Async/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",

```

Comment:
They provide no value outside NuGet packages AFAIK. Anyway, if we're keeping them, they need sensible values.